plugin/src/lib.rs

```diff
@@ -724,6 +724,33 @@ fn fold_first_block(block: P<Block>, m: &mut MutatorPlugin) -> P<Block> {
                         .unwrap(),
                 );
             }
+            for (ref key, ref values) in interchangeables {
+                for value in values.iter() {
+                    let n = *current_count;
+                    let key_ident = key.to_ident();
+                    let value_ident = value.to_ident();
+                    add_mutations(
+                        cx,
+                        mutations,
+                        current_count,
+                        block.span,
+                        &[
+                            &format!(
+                                "exchange {} with {}",
+                                key_ident.as_str(),
+                                value_ident.as_str()
```

Comment:
I get an error during compilation. How can I resolve this error?
```
error[E0599]: no method named `as_str` found for type `syntax::ast::Ident` in the current scope
   --> src/lib.rs:740:43
    |
740 |                                 key_ident.as_str(),
    |                                           ^^^^^^

error[E0599]: no method named `as_str` found for type `syntax::ast::Ident` in the current scope
   --> src/lib.rs:741:45
    |
741 |                                 value_ident.as_str()
    |                                             ^^^^^^
```

Reply:
I think you can write:

```rust
                                key.as_str(),
                                value.as_str()
```

The reason is that `key_ident` is of type `Ident`, which does not have the `as_str` method. On the other hand, `key` is of type `Symbol`, which has the `as_str` method.

Reply:
Thanks @gnieto .

I've changed the code as below and yet to update it.


```
for (ref key, ref values) in interchangeables {
                for value in values.iter() {
                    let n = *current_count;
                    let key_ident = key.to_ident();
                    let value_ident = value.to_ident();
                    add_mutations(
                        cx,
                        mutations,
                        current_count,
                        block.span,
                        &[&format!("exchange {} with {}", key.as_str(), value_ident)],
                    );
                    pre_stmts.push(
                        quote_stmt!(cx,
                        if ::mutagen::now($n) { 
                            let ($key_ident, $value_ident) = ($value_ident, $key_ident)
                         }).unwrap(),
                    );
                }
            }
```

does it looks good?

Reply:
Looks good to me

Reply:
Thanks @gnieto . Currentlt tests in `plugin/` are failing but not due to this PR as tests are failing in master branch too. 
I will wait for @llogiq to approve this PR